Civi/Api4/ContributionProduct.php

```diff
@@ -15,6 +15,7 @@
 /**
  * Contribution Product entity.
  *
+ * @since 5.40
```

Comment:
Thanks @seamuslee001. Should these be 5.41 ?

Reply:
Thanks good point @demeritcowboy updated now